CHANGES.md

```diff
@@ -15,13 +15,21 @@ To be released.
     `IEquatable<Bencodex.Types.Dictionary>` interface.  [[#51]]
  -  Added `Bencodex.Types.ValueKind` enum type.  [[#50], [#53]]
  -  Bencodex lists and dictionaries now can offload their elements:
+     -  Added `Codec.Encode(IValue, IOffloadOptions?)` overloaded method.
+        [[#55]]
+     -  Added `Codec.Encode(IValue, Stream, IOffloadOptions?)` overloaded
+        method.  [[#55]]
+     -  Added `Codec.Decode(byte[], IndirectValue.Loader?)` overloaded method.
+        [[#55]]
+     -  Added `Codec.Decode(Stream, IndirectValue.Loader?)` overloaded method.
+        [[#55]]
      -  Added `Bencodex.Types.IndirectValue` struct.  [[#52]]
      -  Added `List(IEnumerable<IndirectValue>, IndirectValue.Loader)`
         constructor.  [[#52]]
      -  Added `Dictionary(IEnumerable<KeyValuePair<IKey, IndirectValue>>,
         IndirectValue.Loader)` constructor.  [[#52]]
-     -  Added `List.EnumerateIndirectValues()` method.  [[#53]]
-     -  Added `Dictionary.EnumerateIndirectValues()` method.  [[#53]]
```

Comment:
Is these lines remove OK?

Reply:
Yes, I reversed my decision (made in #53) to expose them, because `Codec` supports offloading.